lib/v2/screens/explore_screens/invite/interactor/viewmodels/invite_state.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/datasource/remote/model/balance_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// --- STATE
+class InviteState extends Equatable {
+  final PageState pageState;
+  final RatesState ratesState;
+  final bool isAutoFocus;
+  final String fiatAmount;
+  final BalanceModel? availableBalance;
+  final String? availableBalanceFiat;
+  final bool isCreateInviteButtonEnabled;
+  final double quantity;
+  final bool showAlert;
```

Comment:
lets make these page commands

Reply:
Mm.. I still think it is easier to use a bool, at least in this type of screens (inital false and true if success) If I need to pass data through navigation I just pass the Bloc in order to access the current state.